src/routes/assistants/+page.svelte

```diff
@@ -70,7 +88,21 @@
 				</a>
 			{/if}
 		</div>
-		<h3 class="text-gray-500">Popular assistants made by the community</h3>
+		{#if !data.createdByMe}
+			<!-- if the assistants were created by me, don't show any text -->
+			{#if data.createdByUser}
+				<h3 class="text-gray-500">
+					Assistants created by <a
+						class="hover:underline"
+						href="https://hf.co/{data.createdByUser}"
+						target="_blank">{data.createdByUser}</a
+					>. Click <a class="hover:underline" href="/assistants">here</a> to see all community created
+					assistants
+				</h3>
+			{:else}
+				<h3 class="text-gray-500">Popular assistants made by the community</h3>
+			{/if}
+		{/if}
```

Comment:
```suggestion
		<h3 class="text-gray-500">Popular assistants made by the community</h3>
```

No need to make this text dynamic.